index.js

```diff
@@ -1,14 +1,235 @@
 const express = require('express');
 const helmet = require('helmet');
+const knex = require('knex');
 
 const server = express();
 
 server.use(express.json());
 server.use(helmet());
 
+
+const knexConfig = {
+  client: 'sqlite3',
+  useNullAsDefault: true,
+  connection: {
+      filename: './data/lambda.sqlite3'
+  },
+  debug: true
+}
+
+const db = knex(knexConfig)
+
 // endpoints here
 
+
+// Endpoints for ZOOS table
+
+server.get('/api/zoo', (req, res) => {
+  db('zoos')
+  .then(animals => res.status(200).json(animals))
+  .catch(error => res.status(500).json(error))
+})
+
+server.get('/api/zoo/:id', (req, res) => {
+  const id = req.params.id
+
+  db('zoos')
+  .where({id})
+  .first()
+  .then(animal => {
+    if(!animal) {
```

Comment:
Nice job continuing to use conditionals to send the correct status and information.